src/config/config.js

```diff
@@ -0,0 +1,3 @@
+export const api_token = {
+  token: "Bearer fyLvFPNHSQUtxJX2sPsq7g",
```

Comment:
Please define only key. remove Bearer. 
because if you want to use just API key somewhere, you will not be able to use it.